src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -51,7 +54,7 @@ def handle_exception(ex):  # pylint: disable=too-many-return-statements
             logger.error("To learn more about --query, please visit: "
                          "https://docs.microsoft.com/cli/azure/query-azure-cli?view=azure-cli-latest")
             return 1
-        if isinstance(ex, (CLIError, CloudError, AzureException)):
+        if isinstance(ex, (CLIError, CloudError, AzureException, HttpResponseError, AzureError)):
```

Comment:
Optional: `HttpResponseError` being a subclass of `AzureError`, you could simply catch `AzureError`

Reply:
Got it, will update in next commit.

---
In reply to: [406869816](https://github.com/Azure/azure-cli/pull/12952#discussion_r406869816) [](ancestors = 406869816)